app/addons/news/components.js

```diff
@@ -11,14 +11,78 @@
 // the License.
 
 import React from "react";
+import Cookies from 'universal-cookie';
+const cookies = new Cookies();
 
-const NewsPage = () => {
+function LoadNewsButton(props) {
```

Comment:
Can you make this a fat arrows function. `const LoadNewsButton => ({showNews, isChecked}`